### ISSUE TITLE
Fix init tests

### ISSUE THREAD

**Issue by rtfeldman** [rtfeldman/node-test-runner#44]
[this](https://github.com/rtfeldman/node-test-runner/blob/e566684f0bf9804bf8812ab0b88e0bc8d7bb6ffd/tests/ci.js#L59-L60) and [this](https://github.com/rtfeldman/node-test-runner/blob/e566684f0bf9804bf8812ab0b88e0bc8d7bb6ffd/tests/ci.js#L69-L70) fail in `ci.js`, but work if you run the same steps manually.

The proximal cause is that for some reason in `ci.js`, `elm-test init` generates a template file that is consistently missing an entire line of code. If I run `elm-test init` in my terminal, the line of code is not missing.

**Comment by benansell** [comment]
I had a quick look at this and think the difference in behaviour is due to the differences in the regex engines being used when attempting a multiline search. The [shelljs](https://github.com/shelljs/shelljs#sedoptions-search_regex-replacement-file_array) version is using javascript replace.

To avoid trying to find a way that a multiline regex would work in the terminal and via shelljs, I reframed the problem as "make the failing test in the template pass". I did consider ignoring it but I didn't see concise way of achieving that with elm-test.
